src/app/cancel-mandate/cancel-mandate.component.html

```diff
@@ -0,0 +1,52 @@
+<div id="banner"></div>
+<main>
+  <div>
+    <biggive-page-section>
+      <div class="heading-wrapper">
+        <biggive-heading
+          colour="tertiary"
+          size="1"
+          align="center"
+          [text]="'Cancel regular giving to ' + mandate.charityName"
+        ></biggive-heading>
+      </div>
+      <p>
+        You are currently giving <strong>{{ mandate.donationAmount | money }}</strong> to
+        <strong>{{ mandate.charityName }}</strong>,
+        on the {{ mandate.schedule.dayOfMonth | ordinal }} of each month.
+        Click below to stop future payments.
+      </p>
+
+      <p>This will not affect any payments that have already been collected or are currently being processed.</p>
+
+      <form [formGroup]="cancellationForm" (ngSubmit)="cancel(mandate)">
```

Comment:
this ngSubmit is not actually used so I should probably remove it. I don't think we can use it with `biggive-button` as it is now as our submit button

Reply:
Done.